### ISSUE TITLE
How can I set the `Fill` color for pie chart ?

### ISSUE THREAD

**Issue by Kerlli** [qax-os/excelize#1904]
**Description**

Hi, Maintainers

How can I set the `Fill` color for each part of pie chart ?

Like:

```go
for idx, row := range [][]interface{}{
        {"Apple", "Orange", "Pear"},
        {2, 3, 3},
    } {
        cell, err := excelize.CoordinatesToCellName(1, idx+1)
        if err != nil {
            fmt.Println(err)
            return
        }
        if err := f.SetSheetRow("Sheet1", cell, &row); err != nil {
            fmt.Println(err)
            return
        }
    }

if err := f.AddChart("Sheet1", "E1", &excelize.Chart{
        Type: excelize.Pie,
        Series: []excelize.ChartSeries{
            {
                Name:       "Amount",
                Categories: "Sheet1!$A$1:$C$1",
                Values:     "Sheet1!$A$2:$C$2",
            },
        },
        Title: []excelize.RichTextRun{
            {
                Text: "Fruit Pie Chart",
            },
        },
    }); err != nil {
        fmt.Println(err)
        return
    }
```

I'm wondering about how to set colors for `Apple`, `Orange` and `Pear` 

Thanks.

**Steps to reproduce the issue:**

**Describe the results you received:**

Each series fill color can only be set by:

```go
excelize.ChartSeries{
	Fill: excelize.Fill{
		Type:    "pattern",
		Color:   []string{"#505050"},
		Pattern: 1,
	},
}
```

**Describe the results you expected:**

**Output of `go version`:**

```text
v1.22
```

**Excelize version or commit ID:**

```text
2.8.1
```

**Environment details (OS, Microsoft Excel™ version, physical, etc.):**

**Comment by xuri** [comment]
Duplicated with #1862.

**Comment by xuri** [comment]
I closed this issue, and keep #1862 to track this feature request, if you have any questions, please let me know to reopen this anytime.

**Comment by xuri** [comment]
We added support for set the data point color for doughnut, pie and 3D pie chart. Please upgrade to the master branch code by `go get github.com/xuri/excelize/v2@master`, and this feature will be released on the next version. Now you can set color for data point of pie chart like this:

```diff
package main

import (
    "fmt"

    "github.com/xuri/excelize/v2"
)

func main() {
    f := excelize.NewFile()
    defer func() {
        if err := f.Close(); err != nil {
            fmt.Println(err)
        }
    }()
    for idx, row := range [][]interface{}{
        {"Apple", "Orange", "Pear"},
        {2, 3, 3},
    } {
        cell, err := excelize.CoordinatesToCellName(1, idx+1)
        if err != nil {
            fmt.Println(err)
            return
        }
        if err := f.SetSheetRow("Sheet1", cell, &row); err != nil {
            fmt.Println(err)
            return
        }
    }
    if err := f.AddChart("Sheet1", "E1", &excelize.Chart{
        Type: excelize.Doughnut,
        Series: []excelize.ChartSeries{
            {
                Name:       "Amount",
                Categories: "Sheet1!$A$1:$C$1",
                Values:     "Sheet1!$A$2:$C$2",
+               DataPoint: []excelize.ChartDataPoint{
+                   {
+                       Index: 0,
+                       Fill: excelize.Fill{
+                           Type:    "pattern",
+                           Color:   []string{"003F5C"},
+                           Pattern: 1,
+                       },
+                   },
+                   {
+                       Index: 1,
+                       Fill: excelize.Fill{
+                           Type:    "pattern",
+                           Color:   []string{"58508D"},
+                           Pattern: 1,
+                       },
+                   },
+                   {
+                       Index: 2,
+                       Fill: excelize.Fill{
+                           Type:    "pattern",
+                           Color:   []string{"BC5090"},
+                           Pattern: 1,
+                       },
+                   },
+               },
            },
        },
        Format: excelize.GraphicOptions{
            OffsetX: 15,
            OffsetY: 10,
        },
        Title: []excelize.RichTextRun{
            {
                Text: "Fruit Doughnut Chart",
            },
        },
        PlotArea: excelize.ChartPlotArea{
            ShowPercent: true,
        },
    }); err != nil {
        fmt.Println(err)
        return
    }
    // Save workbook
    if err := f.SaveAs("Book1.xlsx"); err != nil {
        fmt.Println(err)
    }
}
```
